doc/lua_api.md

```diff
@@ -8336,6 +8338,18 @@ child will follow movement and rotation of that bone.
       * `volumetric_light`: is a table that controls volumetric light (a.k.a. "godrays")
         * `strength`: sets the strength of the volumetric light effect from 0 (off, default) to 1 (strongest)
            * This value has no effect on clients who have the "Volumetric Lighting" or "Bloom" shaders disabled.
+      * `light_intensity` is a table that controls calculation of sun light color.
```

Comment:
Maybe call it `sun_light` like as `volumetric_light` or `ambient_light`? `light_intensity` is a pretty vague name. Or even `sky_light` to make the name more collective. This kind of the light can go from not only the default sun, but from any planets, moons and etc.

Reply:
Renamed.